service/controller/chart/v1/resource/releasemigration/create.go

```diff
@@ -0,0 +1,80 @@
+package releasemigration
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/controller/context/reconciliationcanceledcontext"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/chart-operator/service/controller/chart/v1/key"
+)
+
+// EnsureCreated ensures helm release is migrated from a v2 configmap to a v3 secret.
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cr, err := key.ToCustomResource(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	hasConfigMap, err := r.getHelmV2ConfigMaps(ctx, key.ReleaseName(cr))
```

Comment:
```suggestion
	hasConfigMap, err := r.hasHelmV2ConfigMaps(ctx, key.ReleaseName(cr))
```

Sorry I missed this before but since this returns a bool how about using `has` prefix?

Reply:
Good call.